### DISCUSSION TITLE
Suspended Copilot Access – Anyone Know Why?

### DISCUSSION THREAD

**Discussion by arshadbarves** [zed-industries/zed#20225] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Hey folks, I’m getting this email from GitHub when I try to use Copilot with GitHub Copilot Chat:

On behalf of the GitHub Security team, I want to first extend our gratitude for your continued use of GitHub and for being a valued member of the GitHub community.

Recent activity on your account caught the attention of our abuse-detection systems. This activity included use of Copilot that was indicative of scripted interactions or of an otherwise deliberately unusual or strenuous nature. Due to this, we have placed a suspension on your copilot access.

While I’m unable to share specifics on rate limits, we prohibit all use of our servers for any form of excessive automated bulk activity, as well as any activity that places undue burden on our servers through automated means. Please refer to our Acceptable Use Policies on this topic: https://docs.github.com/site-policy/acceptable-use-policies/github-acceptable-use-policies#4-spam-and-inauthentic-activity-on-github.

Please also refer to our Terms for Additional Products and Features for GitHub Copilot for specific terms: https://docs.github.com/site-policy/github-terms/github-terms-for-additional-products-and-features#github-copilot.

I’m kind of confused, though. I was using a custom build (from the current main branch commit) rather than the release version. Could that have triggered the abuse detection? Anyone else run into something similar?

Thanks for the help!

### Environment

MacBook Pro

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by SomeoneToIgnore** [comment]
1. This is not an issue but a discussion.

2. This company is called Zed, not Copilot, no one here knows anything about others' internals and would not be able to give you an answer.
